libs/sdk-backend-base/src/dummyBackend/index.ts

```diff
@@ -736,6 +738,14 @@ class DummyWorkspaceSettingsService implements IWorkspaceSettingsService {
     setLocale(_locale: string): Promise<void> {
         return Promise.resolve();
     }
+
+    setTimeZone(_timeZoneId: string): Promise<void> {
+        return Promise.resolve();
+    }
+
+    setDateFormat(_formatId: string): Promise<void> {
```

Comment:
Should we change the param name `_formatId` → `_dateFormat`?

As the function name is `setDateFormat`, it would make more sense if the param is named `_dateFormat`.